tests/attestation_cert_apis/enc/enc.cpp

```diff
@@ -308,6 +308,6 @@ OE_SET_ENCLAVE_SGX(
     1,    /* ProductID */
     1,    /* SecurityVersion */
     true, /* Debug */
-    128,  /* NumHeapPages */
+    192,  /* NumHeapPages */
```

Comment:
Although this isn't threaded, snmalloc's class size scheme means slightly higher memory usage in this case than with dlmalloc.